### PR TITLE
fix: fee calculations

### PR DESCRIPTION
**Pull request by rcresnik**

### CODE REVIEW

**Review by kenan-mamedoff — APPROVED** [review]

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift [hunk 1]
```diff
@@ -196,19 +204,24 @@ class SwapInteractor: NSObject, Interactor, SwapViewActions {
...
+                self?.dataStore?.toFee = nil
+                self?.dataStore?.toFeeEth = nil
                 self?.presenter?.presentError(actionResponse: .init(error: error))
             }
             group.leave()
         }
         
         group.notify(queue: .main) { [weak self] in
             self?.calculateAmounts(viewAction: viewAction)
+            LoadingView.hide()
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift:224]: viewcontroller stuff

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift [hunk 2]
```diff
@@ -172,19 +176,23 @@ class SwapInteractor: NSObject, Interactor, SwapViewActions {
     }
     
     func setAmount(viewAction: SwapModels.Amounts.ViewAction) {
+        LoadingView.show()
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift:179]: viewcontroller stuff

File: brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift [hunk 3]
```diff
@@ -222,7 +222,7 @@ class SwapCurrencyView: FEView<SwapCurrencyConfiguration, SwapCurrencyViewModel>
         
         cryptoStack.addArrangedSubview(selectorStackView)
         selectorStackView.snp.makeConstraints { make in
-            make.width.equalTo(ViewSizes.huge.rawValue)
+            make.width.equalTo(ViewSizes.huge.rawValue + 10)
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift:225]: we need to invent a new size

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift (hunk 1 above)
Comment (rcresnik) [reply brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift:224]: yp.. this was a dirty fix.. cause the resposnes were so slooow i didnt rly know if i got em or not

**Review by kenan-mamedoff — APPROVED** [review]

**Pull request merged**